README.md

```diff
@@ -40,28 +45,35 @@ $ python -m example.file
 Progress UploadTask_train:  55%|█████████████████████████████████████████
 ```
 
-## Auto Time Series Example: Upload Data, Train Time Series Forecast Experiment and Predict
-train data path: `examples/data/tsf/iris_train.csv`  
-test data path: `examples/data/tsf/iris_train_test.csv`
+## Example Dataset Path 
+* `examples/data/data` - store the general structure dataset
```

Comment:
store the general dataset just under `examples/data/`

Reply:
done